frontend/src/utils/api.js

```diff
@@ -209,6 +209,7 @@ window.api = {
     const res = [];
     const repo = window.REPOS[repoName];
 
+
```

Comment:
Try not to add extra newlines for non-format related PR.

Reply:
Fixed